Neos.Flow/Classes/Core/Booting/Scripts.php

```diff
@@ -249,16 +259,17 @@ public static function initializeSystemLogger(Bootstrap $bootstrap): void
      * Initialize the exception storage
      *
      * @param Bootstrap $bootstrap
-     * @param array $settings The Neos.Flow settings
      * @return ThrowableStorageInterface
      * @throws FlowException
      * @throws InvalidConfigurationTypeException
      */
-    protected static function initializeExceptionStorage(Bootstrap $bootstrap, array $settings): ThrowableStorageInterface
+    protected static function initializeExceptionStorage(Bootstrap $bootstrap): ThrowableStorageInterface
     {
+        $configurationManager = $bootstrap->getEarlyInstance(ConfigurationManager::class);
+        $settings = $configurationManager->getConfiguration(ConfigurationManager::CONFIGURATION_TYPE_SETTINGS, 'Neos.Flow');
+
         $storageClassName = $settings['log']['throwables']['storageClass'] ?? FileStorage::class;
         $storageOptions = $settings['log']['throwables']['optionsByImplementation'][$storageClassName] ?? [];
-        $renderRequestInformation = $settings['log']['throwables']['renderRequestInformation'] ?? true;
```

Comment:
Is this related? Looks as if #2359 was reverted accidentially in this PR?

Reply:
Yeah, you're right. Sorry, I didn't notice. I tried to fix the revert. It seems fine now.